spec/01-unit/01-db/01-schema/08-targets_spec.lua

```diff
@@ -4,6 +4,25 @@ local certificates = require "kong.db.schema.entities.certificates"
 local upstreams = require "kong.db.schema.entities.upstreams"
 local utils = require "kong.tools.utils"
 
+local function setup_global_env()
```

Comment:
Is there a debug code?

Reply:
I suppose there should have been one, but not found, and I confirmed with @ADD-SP .